araproc/framework/data_visualization.py

```diff
@@ -81,6 +82,8 @@ def plot_waveform_bundle(
             xvals = freqs
             yvals = np.log10(np.abs(spectrum))
```

Comment:
Thanks Marco for making things more dynamic. Now that you have opened this PR, could you kindly make the power axis in terms of **dBm** and frequency in **MHz**? I think 
```suggestion
            xvals = freqs*1e3 # from GHz to MHz
            yvals = 10*np.log10(np.abs(spectrum)**2 / 50 / 1e3) # from mV to dBm
```
will do the job.

Reply:
Accordingly, we can change 
line 44 to
```suggestion
           "freq" : "Frequency (MHz)"
```
line 48 to 
```suggestion
           "freq" : "Spectrum (dBm)"
```